src/Notteknektene/firebase/admin.js

```diff
@@ -55,6 +55,10 @@ export const updateScores = async (
     const isFastest = submission && submission.id === fastestSubmission?.id;
     const points = submission ? calculatePoints(submission, isFastest) : 0;
 
+    if (participantName == "Sigve") {
```

Comment:
```suggestion
    if (participantName == "Vegard") {
```
Worst merge request I've ever seen. Do better